v4.txt

```diff
@@ -113,13 +133,16 @@ Note: for the API to return chunks, you must specify `"return_chunks": true` as
 Endpoint: https://api.jina.ai/v1/classify
 Purpose: zero-shot classification for text or images
 Best for: text or image classification without training
-Request body schema: {"application/json":{"model":{"type":"string","required":false,"description":"Identifier of the model to use. Required if classifier_id is not provided.","options":[{"name":"jina-clip-v1","size":"223M","dimensions":768},{"name":"jina-embeddings-v2-base-en","size":"137M","dimensions":768},{"name":"jina-embeddings-v2-base-es","size":"161M","dimensions":768},{"name":"jina-embeddings-v2-base-de","size":"161M","dimensions":768},{"name":"jina-embeddings-v2-base-fr","size":"161M","dimensions":768},{"name":"jina-embeddings-v2-base-code","size":"137M","dimensions":768},{"name":"jina-embeddings-v3","size":"570M","dimensions":1024}]},"classifier_id":{"type":"string","required":false,"description":"The identifier of the classifier. If not provided, a new classifier will be created."},"input":{"type":"array","required":true,"description":"Array of inputs for classification. Each entry can either be a text object {\"text\": \"your_text_here\"} or an image object {\"image\": \"base64_image_string\"}. You cannot mix text and image objects in the same request."},"labels":{"type":"array of strings","required":true,"description":"List of labels used for classification."}}}
```

Comment:
@Andrei997 I split this up since I saw issues with the format.
When you use "input":[{"text":"..."}] for embedding-v3 then we get a 500.
Even I think this is a bug in our API, we need to cover this at the moment